docs/security.html

```diff
@@ -44,8 +44,26 @@
             <li>keystore: the keystore file that stores the certificate. The keystore file contains the private key of the certificate; therefore, it needs to be kept safely.</li>
             <li>validity: the valid time of the certificate in days.</li>
         </ol>
-        Ensure that common name (CN) matches exactly with the fully qualified domain name (FQDN) of the server. The client compares the CN with the DNS domain name to ensure that it is indeed connecting to the desired server, not the malicious one.</li>
-
+        <br>
+	Note: By default the property ssl.endpoint.identification.algorithim has been set to null. Unless configured otherwise hostname verification will not performed. In order to enable hostname verification the following property will need to be set:
+
+	<pre>	ssl.endpoint.identification.algorithim = HTTPS </pre>
```

Comment:
Typo, should be `algorithm`. No spaces before or after `=` for consistency with the rest of the documentation.